server/chat-commands/info.ts

```diff
@@ -2509,14 +2507,14 @@ export const commands: ChatCommands = {
 				return this.errorReply('Invalid image');
 			}
 		}
-		buf += Utils.html`<br /><p style="margin-left:5px;font-size:9pt;color:gray"><small>(Requested by ${request.name})</small>`;
+		buf += Utils.html`<br /><div class="infobox"><small>(Requested by ${request.name})</small>`;
 		if (request.comment) {
-			buf += Utils.html`<br /><p style="margin-left:5px;font-size:9pt;color:gray">${request.comment}</p>`;
+			buf += Utils.html`<br />${request.comment}</small></div>`;
 		} else {
-			buf += `</small></p>`;
+			buf += `</small></div>>`;
 		}
-		this.addBox(buf);
-		room.update();
+		room.add(`|c| ${request.name}|/raw ${buf}`).update();
+		this.modlog('APPROVESHOW', null, `${request.name} (link: ${request.link})`);
```

Comment:
Do we really need to modlog these? They're already in the roomlog.

Reply:
Modlogging was specifically requested, as it's easier to see there.

Reply:
It spams the modlog so much, though. Maybe what we really need is a separate approval log.

Reply:
Yeah, I'm gonna veto modlog.

Reply:
What they want is a roomlog search. Useful feature to build into `/chatlog`. But I don't want it in modlogs.